priv/riak_core.schema

```diff
@@ -200,3 +201,56 @@
     {datatype, {duration, ms}},
     hidden
 ]}.
+
```

Comment:
There are several tests that _could_ be added, but I elected not to because riak_core itself has no schema mappings, only the single cuttlefish translation, which is more effectively tested in other packages' unit tests and in riak_test system-level tests.